04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -319,17 +331,13 @@ RGB_stack_HARV[[1]]
 hist(RGB_stack_HARV,
      maxpixels=ncell(RGB_stack_HARV))
 
```

Comment:
Ok - when i review all of the lessons i'll keep this in mind and will consider cutting it given it may be redundant. thank you for catching that!